reference/5.1/Microsoft.PowerShell.Management/Set-Clipboard.md

```diff
@@ -72,7 +72,7 @@ Get-Content C:\Users\user1\.ssh\id_ed25519.pub | Set-Clipboard
 
 ### -Append
 
-Indicates that the cmdlet does not clear the clipboard and appends content to it.
+Indicates that the cmdlet doesn't clear the clipboard and appends content to it.
```

Comment:
Nit: I wonder if we should make this more obvious by replacing this line with something like:

```suggestion
Indicates that the cmdlet should add to the clipboard instead of replacing it. By default, the
cmdlet clears the current content from the clipboard and sets it to the new content. When this
parameter is specified, the cmdlet appends the new content after a newline to the current content.
```